build.gradle.kts

```diff
@@ -101,59 +101,49 @@ sonar {
+                "dataland-frontend/src/components/resources/frameworkDataSearch/lksg/LksgDataModel.ts," +
+                "dataland-frontend/src/components/resources/frameworkDataSearch/p2p/P2pDataModel.ts," +
+                "dataland-frontend/src/components/resources/frameworkDataSearch/sme/SmeDataModel.ts," +
 
                 // backend
-                "dataland-backend/src/main/kotlin/db/migration/V1_1__CreateBackendTables.kt," +
-                "dataland-frontend/src/frameworks/sfdr/ViewConfig.ts," +
-                "dataland-frontend/src/frameworks/esg-questionnaire/UploadConfig.ts," +
-                "dataland-frontend/src/frameworks/sfdr/UploadConfig.ts",
+                "dataland-backend/src/main/kotlin/db/migration/V1_1__CreateBackendTables.kt",
         )
         property(
             "sonar.exclusions",
 
             // frontend components
             "dataland-frontend/src/components/general/SlideShow.vue," +
-                "dataland-frontend/src/components/resources/newLandingPage/TheQuotes.vue," +
-                "dataland-frontend/src/components/resources/aboutPage/TheAboutIntro.vue," +
-                "dataland-frontend/src/components/resources/aboutPage/TheAboutPrinciples.vue," +
                 "dataland-frontend/src/components/pages/AboutPage.vue," +
                 "dataland-frontend/src/components/generics/TheNewHeader.vue," +
+                "dataland-frontend/src/components/generics/TheNewFooter.vue," +
                 "dataland-frontend/src/components/resources/aboutPage/**," +
                 "dataland-frontend/src/components/resources/newLandingPage/**," +
-                "dataland-frontend/src/components/pages/AboutPage.vue," +
-                "dataland-frontend/src/components/generics/TheNewFooter.vue," +
-
-                // frontend fixtures
-                "dataland-frontend/tests/e2e/fixtures/frameworks/esg-questionnaire/EsgQuestionnaireDataFixtures.ts," +
 
                 // frontend configs
                 "dataland-frontend/src/frameworks/heimathafen/ViewConfig.ts," +
-                // no need to cover the heimathafen-ViewConfig because there are no custom fields
+                // -> no need to cover the heimathafen-ViewConfig because there are no custom fields
 
                 // backend
                 "dataland-backend/src/main/kotlin/," +
                 "dataland-backend/src/main/kotlin/org/dataland/datalandbackend/frameworks/**," +
-                "org/dataland/datalandbackend/model/enums/eutaxonomy/nonfinancials/Activity.kt," +
-                "dataland-frontend/src/components/generics/TheNewFooter.vue" +
-                "dataland-frontend/tests/e2e/fixtures/frameworks/sfdr/SfdrDataFixtures.ts",
+                "org/dataland/datalandbackend/model/enums/eutaxonomy/nonfinancials/Activity.kt" +
+
+                // toolbox
+                "dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/eutaxonomynon" +
```

Comment:
Why is this excluded from sonar analysis?

Reply:
I have no idea.  Removed this exclusion.